src/main/java/edu/iu/terracotta/controller/app/SubmissionController.java

```diff
@@ -94,6 +107,71 @@ public ResponseEntity<SubmissionDto> getSubmission(@PathVariable("experiment_id"
     }
 
 
+    @RequestMapping(value = "/{experiment_id}/conditions/{condition_id}/treatments/{treatment_id}/assessments/{assessment_id}/submissions/new_submission",
+            method = RequestMethod.POST)
+    public ResponseEntity<SubmissionDto> createNewSubmission(@PathVariable("experiment_id") Long experimentId,
```

Comment:
I think we should just update the existing postSubmission method instead of creating a second POST method here. I searched the code and don't see anywhere that the existing postSubmission method is every called so I think it is okay to just replace it.